starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,57 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = 0;
+}
+
+Chronometer.prototype.startClick = function () {
+    var that = this;
+    setInterval(function () {
+        console.log(that.currentTime);
+        that.currentTime++;
+    }, 1000);
+};
+
+Chronometer.prototype.setMinutes = function () {
+
+    var minute = Math.floor(this.currentTime / 60);
+    return minute;
+};
+
+Chronometer.prototype.setSeconds = function () {
+    return this.currentTime % 60;
+};
+
+Chronometer.prototype.twoDigitsNumber = function (value) {
+    if (value < 10) {
+        return "0" + value;
+    } else {
+        return "" + value;
+    }
+};
+
+Chronometer.prototype.setTime = function() {
+    var that = this;
+    this.startClick = setInterval(function(){
```

Comment:
Instead of starting a new interval here, you should just call the `setTime` method in the original interval which is defined in `startClick` method